source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnSetRenderer.mm

```diff
@@ -178,13 +174,18 @@ - (UIView* )render:(UIView<ACRIContentHoldingView> *)viewGroup
                 NSLog(@"unexpected column width property is given");
             }
         }
+
+        if (curView.hasStretchableView || (columnSetElem->GetHeight() == HeightType::Stretch)) {
```

Comment:
THis is a huge function; be nice to have some comments here esp around the move?